app/models/sign_in/session_container.rb

```diff
@@ -41,5 +41,12 @@ def initialize(session:, # rubocop:disable Metrics/ParameterLists
     def persisted?
       false
     end
+
+    def context
+      access_token.to_s.merge({
```

Comment:
What I mean here, is I think all of the individual fields that we are reporting in the 'session container' context should be specified here, like broken out of the `access_token.to_s` into individual fields, since not all of them necessarily relate for the `session container` context